theano/gof/op.py

```diff
@@ -512,7 +528,8 @@ def make_node(self, *inputs):
         MethodNotDefined : the subclass does not override this method.
```

Comment:
In the docstring above it says make_node is mandatory,  but this is no longer the case.